js/utils/selectize.js

```diff
@@ -0,0 +1,15 @@
+/*
+  A wrapper for the selectize jquery element. A tweaked version of it is in the lib folder.
+*/
+
+export const tagsDelimiter = '<=!=>';
```

Comment:
When using selectize for tagging, you can trigger it from an input[type=text] element and, by default, you populate any existing tags inside a value attribute, e.g. `value="tag1,tag2,tag3"`. I found the comma there could be problematic as a separator because certain tags like categories may potentially contain commas. Our client doesn't allow them, but since the server allows them, they might get in there.

So, I've updating the selectize lib to use `<=!=>` as the delimiter, instead of a comma, e.g. `value="tag1<=!=>tag2<=!=>tag3<=!=>`.

When you're dealing with the data, you'll often need to Array.split() and String.join(). Make sure you import this delimiter form here and pass that in to each of those methods.

Reply:
FWIW, I've subsequently found there's no reason you HAVE to use an input[type=text]. You are free to use a \<select\> which allows you to embed the options in markup as option elements, or pass them in via code when initializing the jQuery plugin. <--- might be a slightly easier workflow, but definitely has a more robust api if you have to do something fancy.